cypress/integration/helipopper.spec.ts

```diff
@@ -105,4 +105,17 @@ describe('@ngneat/helipopper', () => {
         .should('exist');
     });
   });
+
+  
+  describe('isVisible', () => {
```

Comment:
Please move it to the is visible spec file, and change the file/dir name to dash case

Reply:
Done